form.php

```diff
@@ -1,3 +1,50 @@
+<?php
+require_once 'functions.php';
+
+$database = getDB();
+
+if (isset($_GET['message']) && $_GET['message'] == 2) {
+    echo 'Incorrect submission, try again';
+}
+
+$artist = '';
+$paintingName = '';
+$year = '' ;
+$image = '';
+
+if(isset($_POST['artist'])
```

Comment:
ok this time, but could have potentially made the isset checking and strlen checking into their own function - then it would have been unit testable